src/src/com/microsoft/aad/adal/ProfileInfo.java

```diff
@@ -18,26 +18,61 @@
 
 package com.microsoft.aad.adal;
 
+import java.util.HashMap;
+
+import android.util.Base64;
+
 class ProfileInfo {
+    static final String TAG = ProfileInfo.class.getSimpleName();
+
+    String mVersion;
```

Comment:
can we leave it for now? The class needs more refactor, but not related to the fix.